samples/shopify/README.md

```diff
@@ -1,53 +1,71 @@
+
+Adds a custom UI field to Contentful that allows users to search and select Shopify products.
+
+![figure](http://contentful.github.io/extensions/assets/Shopify-Product-Demo.gif "Contentful Shopify Product UI Extension demo")
+
+To make the UI extension work you'll need a valid [Storefront API token](https://help.shopify.com/api/reference/access/storefrontaccesstoken) with `unauthenticated_read_product_listings` scope and the API endpoint of your Shopify store. By default the UI-extension uses instance parameters which are pre-configured for demo purposes:
+
+See `extension.json` for the details:
+
+```json
+{
+  "name": "Shopify Products",
+  "id": "contentful-ui-shopify",
+  "fieldTypes": ["Object"],
+  "srcdoc": "./index.html",
+  "parameters": {
+    "instance": [
+      {
+        "id": "apiToken",
+        "name": "Shopify Access Token",
+        "type": "Symbol",
+        "default": "b5392a6d4ba10c47cb4ad1645bf78eb4"
+      },
+      {
+        "id": "apiEndpoint",
+        "name": "Shopify GraphQL endpoint",
+        "type": "Symbol",
+        "default": "https://contentful-demo.myshopify.com/api/graphql"
+      }
+    ]
+  }
+}
 ```
 
-### Configure
+![figure](http://contentful.github.io/extensions/assets//shopify-parameters.jpg "Contentful Shopify UI Extension parameters")
 
-Create a configuration file with your credentials for Contentful.
+## Assign to a field of a content type
 
-```sh
-cp env.example .env
-```
+![figure](http://contentful.github.io/extensions/assets//shopify-object.png "Contentful Shopify Product UI Extension json object")
 
-Open `.env` in a editor of your liking and add your Contentful space ID, and management token. [Learn how to obtain a token](https://www.contentful.com/developers/docs/references/authentication/#getting-an-oauth-token).
+## Usage
 
-Load environment variables
+Make sure you have the Contentful CLI installed:
 
-```sh
-source .env
+```bash
+npm install contentful-cli -g
 ```
 
-### Create
+You are logged in:
 
-```sh
-npm run create
+```bash
+contentful login
 ```
 
-The `create` command will register the extension in your space on Contentful. To use it create a new field of type `JSON object` in the content model of your choice. In the "appearance" tab of this field you can then edit the Shopify settings.
+And have a space selected:
 
-To make the UI extension work you'll need a valid API token and the API endpoint of your Shopify store. By default the UI-extension uses a Contentful shopify store.
-
-![figure](./shopify-parameters.jpg "Contentful Shopify UI Extension parameters")
-
-### Update
-
-```sh
-npm run update
+```bash
+contentful space use
 ```
 
-Update task will upload the extension to your space on Contentful.
+To install the UI Extension:
 
-## Ready to Use
+```bash
+contentful extension create
+```
 
-![figure](./shopify-object.png "Contentful Shopify Product UI Extension json object")
\ No newline at end of file
+To update the UI Extension:
```

Comment:
Consistency nitpick: A new line is missing between paragraph and code snippet.

Reply:
done